CONTRIBUTING.md

```diff
@@ -8,28 +8,29 @@ Ready to contribute? Here's how to set up `aicsimageio` for local development.
 
 1. Fork the `aicsimageio` repo on GitHub.
 
-1. Clone your fork locally:
+2. Clone your fork locally:
```

Comment:
Why was this numbering changed?

Reply:
While it doesn't matter for the documentation _render_, it looks better in an IDE when editing / following along locally. It was a minor nitpick by someone that I agree with.